build.gradle

```diff
@@ -11,9 +11,9 @@ buildscript {
         ]
 
         libraries = [
-                result               : "com.github.kittinunf.result:result:5.2.0",
-                retrofit             : "com.squareup.retrofit2:retrofit:2.9.0",
-                retrofitConverterGson: "com.squareup.retrofit2:converter-gson:2.9.0",
+                result               : "com.github.kittinunf.result:result:5.6.0",
+                retrofit             : "com.squareup.retrofit2:retrofit:2.12.0",
+                retrofitConverterGson: "com.squareup.retrofit2:converter-gson:2.12.0",
```

Comment:
Upgrading some dependencies likte Retrofit forces Kotlin 2.1, postponed for now.